### PR TITLE
update labels

### PR DESCRIPTION
**Pull request by agebhar1**
Due to #446 beta/WIP projects should be labeled by `:construction:` or
via UTF-8 character 🚧.

### CODE REVIEW

**Comment by devhkr** [comment]
Isn't it better using ```:construction:```? Otherwise, I only see ```🚧``` in my (web)editor. It's also clear to see what icon is used. What do you think?

**Comment by agebhar1** [comment]
I agree with you @devhkr. One may update/change all UTF-8 labels by there equivalent shortname in another request.

File: README.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
...
 * [Conduit](https://github.com/ehazlett/conduit) - Experimental deployment system for Docker by [@ehazlett](https://github.com/ehazlett)
 * [depcon](https://github.com/gondor/depcon) - Depcon is written in Go and allows you to easily deploy Docker containers to Apache Mesos/Marathon, Amazon ECS and Kubernetes.  By [@gonodr][gondor]
-* [deploy](https://github.com/ttiny/deploy) 🤕 - Git and Docker deployment tool. A middle ground between simple Docker composition tools and full blown cluster orchestration by [@ttiny](https://github.com/ttiny)
+* [deploy](https://github.com/ttiny/deploy) 🚧 - Git and Docker deployment tool. A middle ground between simple Docker composition tools and full blown cluster orchestration by [@ttiny](https://github.com/ttiny)
```
Comment (devhkr) [inline README.md]: On the other side, the last commit was nearly one year ago and 9 open issues unresolved. Maybe ```:skull:``` is appropriate? What do you think?

Comment (agebhar1) [reply README.md]: Good catch @devhkr I'll update with another commit.

**Comment by veggiemonk** [comment]
@agebhar1 good job 👍

**Comment by agebhar1** [comment]
Thanks @veggiemonk :smiley:

**Pull request merged**